commerce-add-ons/payment-providers/buckaroo/configuring-umbraco.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Configuring Umbraco
+description: >-
+  Learn how to configure the Umbraco backoffice for enabling the use of Buckaroo
+  as a payment method.
+---
+
+# Configure Umbraco
+
+## Step 1: Create Payment Method
+
+The following steps are all handled through the Umbraco backoffice.
+
+1. Navigate to **Settings > Commerce > Stores > {Store Name} > Payment Methods** section.
+2. Select the **Create Payment Method** button to create a new payment method.
+3. Choose **Buckaroo Checkout** from the list of available payment providers.
```

Comment:
Based on the image provided below there is no *Buckaroo Checkout*
```suggestion
3. Choose **Buckaroo One Time Payment** from the list of available payment providers.
```

Reply:
@umbracotrd can you please confirm if the above is indeed correct? 😊

Reply:
Nice catch @alina-tincas. I'm committing your suggestion.